src/Integrations/Mailer/SettingsMailer.php

```diff
@@ -214,157 +226,181 @@ public function getSettingsData(string $formId): array
+									'component' => 'input',
+									'inputName' => $this->getSettingsName(self::SETTINGS_MAILER_SENDER_NAME_KEY),
+									'inputFieldLabel' => \__('Email client sender name', 'eightshift-forms'),
+									'inputFieldHelp' => \__('Most e-mail clients will show this instead of the e-mail address in the list of e-mails. Make this something related to your brand that will distinguish you from the rest.', 'eightshift-forms'),
+									'inputType' => 'text',
+									'inputIsRequired' => true,
+									'inputValue' => $this->getSettingsValue(self::SETTINGS_MAILER_SENDER_NAME_KEY, $formId),
+								],
+								[
+									'component' => 'divider',
+									'dividerExtraVSpacing' => 'true',
+								],
+								[
+									'component' => 'input',
+									'inputName' => $this->getSettingsName(self::SETTINGS_MAILER_SENDER_EMAIL_KEY),
+									'inputFieldLabel' => \__('Email client from email', 'eightshift-forms'),
+									'inputFieldHelp' => \__('Most e-mail clients will us this as field as the senders email and will use it when the user clicks on the respond to button. Make this something like: `info@infinum.com`.', 'eightshift-forms'),
+									'inputType' => 'text',
+									'inputIsEmail' => true,
+									'inputIsRequired' => true,
+									'inputValue' => $this->getSettingsValue(self::SETTINGS_MAILER_SENDER_EMAIL_KEY, $formId),
+								],
 							],
-							...($isSenderUsed ? [
+						],
+						[
+							'component' => 'tab',
+							'tabLabel' => \__('E-mail', 'eightshift-forms'),
+							'tabContent' => [
+								[
+									'component' => 'input',
+									'inputName' => $this->getSettingsName(self::SETTINGS_MAILER_TO_KEY),
+									'inputFieldLabel' => \__('Recipient email', 'eightshift-forms'),
+									// translators: %s will be replaced with forms field name.
+									'inputFieldHelp' => \sprintf(\__('
+										This e-mail address will be used to sent the form data to.<br />
+										You can use multiple emails here by separating them by comma.<br /><br />
+										%s', 'eightshift-forms'), $this->getFieldTagsOutput($fieldNameTags)),
+									'inputType' => 'text',
+									'inputPlaceholder' => 'info@infinum.com',
+									'inputIsRequired' => true,
+									'inputValue' => $this->getSettingsValue(self::SETTINGS_MAILER_TO_KEY, $formId),
+								],
 								[
 									'component' => 'divider',
 									'dividerExtraVSpacing' => 'true',
 								],
 								[
 									'component' => 'input',
-									'inputName' => $this->getSettingsName(self::SETTINGS_MAILER_SENDER_SUBJECT_KEY),
+									'inputName' => $this->getSettingsName(self::SETTINGS_MAILER_SUBJECT_KEY),
 									'inputFieldLabel' => \__('Subject', 'eightshift-forms'),
 									// translators: %s will be replaced with forms field name.
-									'inputFieldHelp' => \sprintf(\__('Use template tags to use submitted form data (e.g. <code>{field-name}</code>)', 'eightshift-forms'), $formNames),
+									'inputFieldHelp' => \sprintf(\__('
```

Comment:
This can just be the `fieldsTagOutput`, the label "Subject" is clear enough.

Reply:
dummy proof